postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -315,6 +315,18 @@ instances:
         #
         # seen_samples_cache_maxsize: 10000
 
+        ## @param explain_parameterized_queries - boolean - optional - default: false
+        ## Note, this is a BETA feature. This option will enable the ability to explain parameterized queries.
+        ## This is useful if your SQL clients are using the extended query protocol.
+        #
+        # explain_parameterized_queries: false
+
+        ## @param max_pg_prepared_statements_space - integer - optional - default: 20
+        ## Prepared statements may be created to enable some functionality, this option configures the max amount
+        ## of space (in megabytes) the integration will allocate.
+        #
+        # max_pg_prepared_statements_space: 20
```

Comment:
Is this space on the agent or the db?

Reply:
The database, I can clarify in the string, thanks!